ocp-config/prov-app/cm.yml

```diff
@@ -128,9 +128,7 @@ objects:
       bitbucket.technical.user=${CD_USER_ID}
       bitbucket.opendevstack.project=opendevstack
 
-      scm.global.readablerepos.opendevstack[0]=ods-jenkins-shared-library
-      scm.global.readablerepos.opendevstack[1]=ods-quickstarters
-      scm.global.readablerepos.opendevstack[2]=ods-mro-jenkins-shared-library
+      scm.global.readablerepos.opendevstack=ods-jenkins-shared-library,ods-quickstarters
```

Comment:
Removing `ods-mro-jenkins-shared-library` as it is no longer needed since it is merged into `ods-jenkins-shared-library`.